kubectl-apply-all.sh

```diff
@@ -12,4 +12,7 @@
 #
 # Apply the base Soucegraph deployment
 # shellcheck disable=SC2068
-kubectl apply --prune -l deploy=sourcegraph -f base --recursive $@
+echo "Printing diff"
```

Comment:
Do we need to change the commands for the old base cluster? These seem roughly equivalent, but if that's the case, I would prefer not changing them in case there are subtle differences or if changing this file causes a merge conflict for anyone who has forked this repository.